### PR TITLE
add copy and paste shortcuts to multiline mode

### PR DESCRIPTION
**Pull request by guhitb**
Added copy and paste keybindings to multiline mode.
- i - focus the textarea
- y - copy last response to clipboard
- p - paste clipboard to textarea

These keybindings are only proccessed while the textarea is unfocused.

It uses this library https://github.com/atotto/clipboard.

Using the shortcuts to copy and paste between tgpt windows:

https://github.com/aandrew-me/tgpt/assets/147658676/29a9e46e-eacc-4f63-9683-f583cc56a62f

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -307,10 +315,35 @@ func (m model) Update(msg tea.Msg) (tea.Model, tea.Cmd) {
...
+				err := clipboard.WriteAll(lastResponse)
+				if err != nil {
+					panic("could not write to clipboard")
+				}
+			case "p":
+				m.textarea.Focus()
+				clip, err := clipboard.ReadAll()
+				msg.Runes = []rune(clip)
+				if err != nil {
+					panic("could not write to clipboard")
```
Comment (aandrew-me) [inline main.go:343]: Could not read from clipboard*

**Comment by aandrew-me** [comment]
Definitely can be very useful, appreciate it

**Comment by aandrew-me** [comment]
Also I noticed that the repo hasn't been updated in a long time, there is another package with more recent updates - https://github.com/golang-design/clipboard
Maybe a better choice?

File: main.go (hunk 1 above)
Comment (guhitb) [reply main.go]: Can you check if pasting works on your machine now? `golang-design/clipboard` says it only depends on X11, the previous library depended on xclip.

**Comment by guhitb** [comment]
Okay, I think I did that but have trouble figuring out git sometimes. Did you mean to merge into one commit, or to remove the change to `golang-design/clipboard` and go back to using `atotto/clipboard`?

**Comment by aandrew-me** [comment]
> Okay, I think I did that but have trouble figuring out git sometimes. Did you mean to merge into one commit, or to remove the change to `golang-design/clipboard` and go back to using `atotto/clipboard`?

Yes go back to using `atotto/clipboard`

**Comment by aandrew-me** [comment]
Here is the content of your old main.go file, I made small changes related to error messages
[main.txt](https://github.com/aandrew-me/tgpt/files/14116964/main.txt)

**Comment by guhitb** [comment]
Alright, I think this is correct now. Thanks for linking that file.
- printing clipboard errors instead of panicing
- back to using `atotto/clipboard`

**Comment by aandrew-me** [comment]
I changed shortcut from `y` to `c`, so that its more intuitive. (c for copy, p for paste)

**Pull request merged**